src/Commands/MigrateCommands.php

```diff
@@ -0,0 +1,169 @@
+   * @var \Drupal\Core\Entity\EntityFieldManagerInterface
+   */
+  protected $entityFieldManager;
+
+  /**
+   * The entity last installed schema repository service.
+   *
+   * @var \Drupal\Core\Entity\EntityLastInstalledSchemaRepositoryInterface
+   */
+  protected $entityLastInstalledSchemaRepository;
+
+  /**
+   * The entity display repository service.
+   *
+   * @var \Drupal\Core\Entity\EntityDisplayRepositoryInterface
+   */
+  protected $entityDisplayRepository;
+
+  /**
+   * The widget plugin manager.
+   *
+   * @var \Drupal\Core\Field\WidgetPluginManager
+   */
+  protected $widgetPluginManager;
+
+  /**
+   * Constructor.
+   *
+   * @param \Drupal\Core\Database\Connection $database
+   *   The database service.
+   * @param \Drupal\Core\Config\ConfigFactoryInterface $configFactory
+   *   The config factory service.
+   * @param \Drupal\Core\KeyValueStore\KeyValueFactoryInterface $keyValue
+   *   The key value service.
+   * @param \Drupal\Core\Entity\EntityFieldManagerInterface $entityFieldManager
+   *   The entity field manager service.
+   * @param \Drupal\Core\Entity\EntityLastInstalledSchemaRepositoryInterface $entityLastInstalledSchemaRepository
+   *   The entity last installed schema repository service.
+   * @param \Drupal\Core\Entity\EntityDisplayRepositoryInterface $entityDisplayRepository
+   *   The entity display repository service.
+   * @param \Drupal\Core\Field\WidgetPluginManager $widgetPluginManager
+   *   The widget plugin manager.
+   */
+  public function __construct(Connection $database, ConfigFactoryInterface $configFactory, KeyValueFactoryInterface $keyValue, EntityFieldManagerInterface $entityFieldManager, EntityLastInstalledSchemaRepositoryInterface $entityLastInstalledSchemaRepository, EntityDisplayRepositoryInterface $entityDisplayRepository, WidgetPluginManager $widgetPluginManager) {
+    $this->database = $database;
+    $this->configFactory = $configFactory;
+    $this->keyValue = $keyValue;
+    $this->entityFieldManager = $entityFieldManager;
+    $this->entityLastInstalledSchemaRepository = $entityLastInstalledSchemaRepository;
+    $this->entityDisplayRepository = $entityDisplayRepository;
+    $this->widgetPluginManager = $widgetPluginManager;
+  }
+
+  /**
+   * Migrates an entity_reference field to entity_reference_override.
+   *
+   * @param string $entity_type_id
+   *   The entity type ID.
+   * @param string $field_name
+   *   The field name.
+   *
+   * @command entity_reference_override:migrate
+   *
+   * @usage drush entity_reference_override:migrate
+   *   Migrates an entity_reference field to entity_reference_override.
+   */
+  public function migrate($entity_type_id, $field_name) {
+    $field_storage_config = $this->configFactory->getEditable("field.storage.$entity_type_id.$field_name");
+
+    if ($field_storage_config->get('type') !== 'entity_reference') {
+      $this->io()->error(\dt('Not an entity reference field'));
+      return;
+    }
+
+    $schema_spec = [
```

Comment:
Might be worth an @see to \Drupal\entity_reference_override\Plugin\Field\FieldType\EntityReferenceOverrideItem::schema() so we know we have to keep these in sync. And adding an @see over there.